t/unit/test_admin.py

```diff
@@ -51,3 +55,20 @@ def test_toggle_action_all_disabled(self):
         self.assertTrue(e1)
         self.assertTrue(e2)
         self.assertTrue(e3)
+
+    def test_validate_unique_raise_cases(self):
+        periodic_task = PeriodicTask()
+        crontab_schedule = CrontabSchedule()
+        interval_schedule = IntervalSchedule()
+
+        with self.assertRaises(ValidationError):
+            periodic_task.validate_unique()
+
+        periodic_task.crontab = crontab_schedule
+        ret_value = periodic_task.validate_unique()
+        self.assertIsNone(ret_value,
+                          'validate_unique should return None')
+
+        periodic_task.interval = interval_schedule
+        with self.assertRaises(ValidationError):
+            periodic_task.validate_unique()
```

Comment:
i think it would be useful to add separate test cases for each type:
```
from itertools import permutations

def test_validate_unique_raises_if_schedule_not_set(self):
    with self.assertRaises(ValidationError):
        PeriodicTask().validate_unique()

def test_validate_unique_raises_if_for_multiple_schedules(self):
    schedules = [
        ('crontab', CrontabSchedule()),
        ('interval', IntervalSchedule()),
        ('solar', SolarSchedule()),
    ]
    for options in permutations(schedules, 2):
        with self.assertRaises(ValidationError):
            PeriodicTask(**options).validate_unique()

def test_validate_unique_not_raises(self):
    PeriodicTask(crontab=CrontabSchedule()).validate_unique()
    PeriodicTask(interval=IntervalSchedule()).validate_unique()
    PeriodicTask(solar=SolarSchedule()).validate_unique()
```